docs/user-guide/faq.md

```diff
@@ -144,7 +144,7 @@ All these patterns disallow CSS identifiers that start with a digit, two hyphens
 
 Use the [`defaultSeverity`](configuration.md#defaultseverity) configuration option.
 
-## Can I bundle more than one sharable config within a npm package?
+## Can I bundle more than one sharable config within an npm package?
```

Comment:
This is correct. Although "an" is usually used before a word beginning with a vowel, it is also used before any word beginning with a "vowel sound". Since "npm" is pronounced "en pee em" this is right.